frontend/src/components/periodical-evaluation/references/PeriodicalEditionReferenceSummary.vue

```diff
@@ -1,11 +1,47 @@
 <script lang="ts" setup>
+import { storeToRefs } from "pinia"
+import { computed, watch } from "vue"
 import DecisionSummary from "@/components/DecisionSummary.vue"
 import { DisplayMode } from "@/components/enumDisplayMode"
+import { PublicationState } from "@/domain/publicationStatus"
 import Reference from "@/domain/reference"
+import useSessionStore from "@/stores/sessionStore"
 
 const props = defineProps<{
   data: Reference
 }>()
+const { user } = storeToRefs(useSessionStore())
+
+const linkClickable = computed(() => {
+  const docUnit = props.data.documentationUnit
+  const userDocOffice = user.value?.documentationOffice?.abbreviation
+
+  const isOwningDocOffice =
+    docUnit?.documentationOffice?.abbreviation === userDocOffice
+  const isCreatingDocOffice =
+    docUnit?.creatingDocOffice?.abbreviation === userDocOffice
+
+  const status = docUnit?.status?.publicationStatus
+  const isPublishedOrInPublishing =
+    status === PublicationState.PUBLISHED ||
+    status === PublicationState.PUBLISHING
+  const isPendingAndCreatingDocOffice =
+    status === PublicationState.EXTERNAL_HANDOVER_PENDING && isCreatingDocOffice
+
+  return (
+    isOwningDocOffice ||
+    isPublishedOrInPublishing ||
+    isPendingAndCreatingDocOffice
+  )
+})
+
```

Comment:
@leonie-koch ich frage mich, ob diese Logik im Frontend oder im Backend besser aufgehoben ist. Die meisten Entscheidungen rund um Autorisierung sind meine ich aktuell im Backend (z.B. [isEditable, isDeletable in der DocUnit](https://github.com/digitalservicebund/ris-backend-service/blob/a3b010ef8a3c2313a6025f2fce6f200361f23db4/backend/src/main/java/de/bund/digitalservice/ris/caselaw/domain/DocumentationUnit.java#L46-L47)) . Vielleicht könnte man auch das Feld `referenceFound` verwenden 🤔

Reply:
Du hast völlig recht. Ich nehme mir das mal mit für heute nachmittag und würde dieses ticket trotzdem schonmal durchschieben in done